### PR TITLE
hot fix: hide token empty data

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- limit order: fix wrong decimals issues
- hot fix: hide all token decimals=0 in token search modal

### CODE REVIEW

File: src/hooks/Tokens.ts [hunk 1]
```diff
@@ -240,16 +240,19 @@ export const fetchTokenByAddress = async (address: string, chainId: ChainId) =>
...
   const response = await axios.get(`${KS_SETTING_API}/v1/tokens?addresses=${address}&chainIds=${chainId}`)
   const tokens = response?.data?.data?.tokens ?? []
-  return tokens.map(formatAndCacheToken)
+  return filterTruthy(tokens.map(formatAndCacheToken)) as WrappedTokenInfo[]
 }
 
 export const formatAndCacheToken = (tokenResponse: TokenInfo) => {
   try {
     const tokenInfo = new WrappedTokenInfo(tokenResponse)
+    if (!tokenInfo.decimals) {
```
Comment (viet-nv) [inline src/hooks/Tokens.ts:249]: what about decimal 0?

Comment (nguyenhoaidanh) [reply src/hooks/Tokens.ts:249]: 0 => hide

Comment (viet-nv) [reply src/hooks/Tokens.ts:249]: why? it should be valid

Comment (nguyenhoaidanh) [reply src/hooks/Tokens.ts]: updated. missing condition, nice catch 💯 
```
 if (!tokenInfo.decimals && !tokenInfo.symbol && !tokenInfo.name) {
      return
    }
```

**Review by viet-nv — APPROVED** [review]

**Review by viet-nv — APPROVED** [review]

**Pull request merged**